src/main/java/io/hops/util/featurestore/FeaturestoreHelper.java

```diff
@@ -798,18 +798,21 @@ public static Boolean validatePrimaryKey(Dataset<Row> featuregroupDf, String pri
   public static void validateMetadata(String name, Tuple2<String, String>[] dtypes,
                                       List<String> dependencies, String description) {
     Pattern namePattern = Pattern.compile("^[a-zA-Z0-9-_]+$");
-    if (name.length() > 256 || name.equals("") || !namePattern.matcher(name).matches())
+    if (name.length() > 256 || name.equals("") || !namePattern.matcher(name).matches() || name.contains("-"))
```

Comment:
Same thing for the Hopsworks PR. If you remove the last `-` from the pattern above, the REGEX will make sure that names don't contain `-`

Reply:
Oops my bad, I checked regex to quick, I thought regex already didn't include it as it is from Hive docs but for some reason the checks failed anyway that's why I added the extra check. Will fix in hops-util-py too

Reply:
Fixed